packages/engine/src/physics/components/ColliderComponent.ts

```diff
@@ -14,6 +14,8 @@ export class ColliderComponent extends Component<ColliderComponent> {
   type: string
   mass: number
   position: Vector3
+  velocityPositionsToCalculate = 4;
```

Comment:
these are not needed, seems like a merge error

Reply:
Ya I don't know how collidercomponent got changed